src/modules/products/entities/product.entity.ts

```diff
@@ -212,4 +213,7 @@ export class Product {
 
   @OneToMany(() => Wishlist, (wishlist) => wishlist.product)
   wishlistEntries: Wishlist[];
+
+  @ManyToMany(() => Order, (order) => order.products)
+  orders: Order[];
```

Comment:
Also we need a field isAvailable here. And it should be false when the product is rented.

Reply:
done